lib/pbench/test/unit/server/conftest.py

```diff
@@ -1025,8 +1037,9 @@ def generate_api_key(
         "iat": current_utc,
         "user_id": user.id,
         "username": user.username,
+        "name": name,
```

Comment:
>But, since the server code doesn't do this, I don't see why the unit test should do it...

I added this only for unit tests, because in order to test that a single user can have multiple keys effectively. The issue is  I couldn't create a multiple keys for a single, it returns a `Duplicate api_key` error because other fields like `current_utc`,`user.id`,`user.username` won't change for a single user. That's is the reason I had included the `name` field into the payload